packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/AccordianLogs.js

```diff
@@ -74,3 +86,11 @@ export default function AccordianLogs(props: AccordianLogsProps) {
     </div>
   );
 }
+
+AccordianLogs.defaultProps = {
+  interactive: true,
```

Comment:
what is the meaning of the "interactive" flag?

Reply:
When set to `false` the component is not interactive.